examples/basic/server.js

```diff
@@ -1,6 +1,6 @@
 const express = require('express')
-
-const { ApolloVoyagerServer, gql } = require('../../packages/apollo-voyager-server')
+const { LOCAL } = process.env; //use this environment variable to run with code from local repository not the published version
+const { ApolloVoyagerServer, gql } = LOCAL ? require('../../packages/apollo-voyager-server') : require("@aerogear/apollo-voyager-server");
```

Comment:
Ah.. We do not need that anymore. `"@aerogear/apollo-voyager-server"` will work and we have this on master.

Reply:
removed